.github/actions/publish_to_pages_production/action.yml

```diff
@@ -0,0 +1,12 @@
+name: publish_to_pages_production
+description: Publish to cloudflare pages (production)
```

Comment:
I think we need to set environment variables for cloudflare deployment, because it needs to use `CLOUDFLARE_API_TOKEN` and `CLOUDFLARE_ACCOUNT_ID` for credentials:
```
env:
    CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
    CLOUDFLARE_ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
```

Reply:
Here's the docs for reference: https://developers.cloudflare.com/workers/wrangler/ci-cd/